src/lib/MiradorViewer.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import { v4 as uuid } from 'uuid';
-import { App } from '../components/App';
+import HotApp from '../components/App';
```

Comment:
I guess exporting a HotApp for a production build has a minimal footprint so it shouldn't be an issue?

https://github.com/gaearon/react-hot-loader#what-about-production

Reply:
That's the claim 🤷‍♂️